audformat/core/utils.py

```diff
@@ -212,6 +212,72 @@ def concat(
         return df
 
 
+def duration(
+        obj: typing.Union[pd.Index, pd.Series, pd.DataFrame],
+        *,
+        root: str = None,
+        num_workers: int = 1,
+        verbose: bool = False,
+) -> pd.Timedelta:
+    r"""Total duration of all entries present in the object.
+
+    The object might contain a segmented or a filewise index.
+    For a segmented index the duration is calculated
+    from its start and end values.
+    If an end value is ``NaT``
+    or the object contains a filewise index
+    the duration is calculated from the media file
+    by calling :func:`audiofile.duration`.
+
+    Args:
+        obj: object conform to
+            :ref:`table specifications <data-tables:Tables>`
+        root: root directory under which the files referenced in the index
+            are stored.
+            Only relevant when the duration of the files
+            needs to be detected from the file.
+        num_workers: number of parallel jobs.
+            Only relevant when the duration of the files
+            needs to be detected from the file.
+            If ``None`` will be set to the number of processors
+            on the machine multiplied by 5
```

Comment:
I always unsure whether we should end with a `.` or not. Here you don't, but for `root` and `verbose` you do. Maybe we should try to stick to one rule everywhere?

Reply:
Yes, sorry for that.
The rule we have followed (let's say most of the time) is no `.` at the end.

I removed the `.` at the end of the sentences.